python/device/device.py

```diff
@@ -102,7 +102,7 @@ def custom_props(extra_props):
 
 class CustomPropertyNames(object):
```

Comment:
Why are these values here instead of in the constants file?

Reply:
I looked at how these are used, and we don't need these anymore.  I've removed them.